packages/patternlab/styleguide/source/assets/js/modules/accordions.js

```diff
@@ -1,11 +1,28 @@
 import checkActive from "../helpers/cssControlCode.js";
 
 export default (function (window,document,$,undefined) {
+  let $jsAccordion = $('.js-accordion');
 
-  $('.js-accordion').each(function(index){
+
+  $jsAccordion.each(function(index){
     init.apply(this, [index]);
   });
 
+  // In case that all paragraphs are expanded by default, change the "expand all" link to "collapse all".
+  if($(".ma__collapsible-content--extended").length) {
+    let allExpanded = true;
+
+    $('.ma__collapsible-content--extended .js-accordion').each(function(index){
+      if(! $(this).hasClass('is-open')) {
+        allExpanded = false;
```

Comment:
I think you might also want to add this to the individual accordion toggle function. As, if the user manually expand / collapse all the accordions, the state is wrong. e.g.
![Screen Shot 2021-09-14 at 12 29 39 PM](https://user-images.githubusercontent.com/5789411/133297713-e2bf1a1b-71bc-4b89-9bbc-eb7cd636332c.png)